File: core/src/migration_example.rs
pub fn greet(name: &str) -> String {
    format!("Hello, {}", name)
}

pub fn add(a: i32, b: i32) -> i32 {
    a + b
}

pub fn format_user(id: i32, username: &str) -> String {
    format!("{}:{}", id, username.to_lowercase())
}

File: out2/src/src/migration_example.py
from typing import Optional

class User:
    def __init__(self, id: int, username: str):
        self.id = id
        self.username = username

def greet(name: str) -> str:
    return f"Hello, {name}"

def add(a: int, b: int) -> int:
    return a + b

def format_user(id: int, username: str) -> str:
    return f"{id}:{username.lower()}"